### PR TITLE
feat(cdc): rate limit for cdc backfill

### PR DESCRIPTION
**Pull request by StrikeW**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Support `alter table t1 set backfill_rate_limit = xx` for cdc table.
- Extend `update_mv_rate_limit_by_table_id` to modify the `rate_limit` of `StreamCdcScan` node
 
Tested locally.

#17913 

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [x] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

Support altering `backfill_rate_limit` of cdc table.

</details>

### CODE REVIEW

File: src/meta/src/manager/catalog/fragment.rs [hunk 1]
```diff
@@ -1079,6 +1079,11 @@ impl FragmentManager {
                 for actor in &mut fragment.actors {
                     if let Some(node) = actor.nodes.as_mut() {
                         visit_stream_node(node, |node_body| match node_body {
+                            // rate limit for cdc backfill
+                            NodeBody::StreamCdcScan(ref mut node) => {
```
Comment (kwannoel) [inline src/meta/src/manager/catalog/fragment.rs:1083]: This part looks correct, but the fragment_mask in L1075 above seems to be not set for `StreamCdcScan`.

**Review by kwannoel — COMMENTED** [review]
Thanks for your work! Could you add e2e_source_test for cdc backfill?

You can reuse the test logic under https://github.com/risingwavelabs/risingwave/blob/main/e2e_test/source/basic/alter/rate_limit_table_kafka.slt.

See that we can set `cdc_backfill` to 0 initially, and alter it to resume.

**Comment by kwannoel** [comment]
Rest LGTM

File: src/meta/src/manager/catalog/fragment.rs (hunk 1 above)
Comment (chenzl25) [reply src/meta/src/manager/catalog/fragment.rs:1083]: I found it in the comment. 
> /// Include StreamScan and StreamCdcScan
    StreamScan = 16,

**Review by chenzl25 — APPROVED** [review]
LGTM

File: src/meta/src/manager/catalog/fragment.rs (hunk 1 above)
Comment (kwannoel) [reply src/meta/src/manager/catalog/fragment.rs:1083]: Seems like it should be separate. We can fix it in a separate PR. They are fundamentally different kinds of scans.

**Review by kwannoel — APPROVED** [review]
LGTM, thanks!

**Pull request merged**